apidocs/index.html

```diff
@@ -74,7 +74,7 @@
   
   // Build a system
   const ui = SwaggerUIBundle({
-    url: "http://api.susi.ai/docs/swagger.json",
```

Comment:
I debugged the `https://dev.susi.ai/apidocs/` page and found out that the error message was "mixed content". So this does not complain about CORS but only that the endpoint is http, not https.

Please use the url `https://api.susi.ai/docs/swagger.json` (added 's') instead the cors.io endpoint as a first try, so we depend not on an external service